var/spack/repos/builtin/packages/opencv/package.py

```diff
@@ -233,10 +234,16 @@ class Opencv(CMakePackage, CudaPackage):
         "js_bindings_generator",
     ]
 
+    # Define the list of libraries objects that may be used
+    # to find an external installation and its variants
+    libraries = []
+
     # module variants
     for mod in modules:
         # At least one of these modules must be enabled to build OpenCV
         variant(mod, default=False, description="Include opencv_{0} module".format(mod))
+        lib = f'libopencv_{mod}'
```

Comment:
Can't use f-strings, we need to support Python 2.7

Reply:
Okay, I just pushed a fix.